### DISCUSSION TITLE
Port to NUCLEO-WL55JC board

### DISCUSSION THREAD

**Discussion by piyushy6** [Lora-net/LoRaMac-node#1041] [Q&A] [answered]
How can we port it to NUCLEO-WL55JC board?

**Comment by mluis1** [comment, marked as answer]
There are already several topics discussing this topic (#1003, #1023 and #1036).
We mark it as duplicate and close this issue.

In the future it would be nice if you could post this kind of questions on the project [Discussions](https://github.com/Lora-net/LoRaMac-node/discussions) tab. It is a better place to engage discussions and then we can agree if it is an issue or not.
If you don't mind I move this issue to the [Discussions](https://github.com/Lora-net/LoRaMac-node/discussions) tab.

**Comment by piyushy6** [comment]
Thanks for letting me know.
